dc/s2n-quic-dc/src/stream/recv/state.rs

```diff
@@ -751,8 +754,52 @@ impl State {
             }
             packet_len => {
                 buffer.truncate(packet_len);
-                // TODO duplicate the transmission in case we have a lot of gaps in packets
+
+                // get how many intervals we're tracking - the more there are, the more loss the network
+                // is experiencing
+                let intervals = self.stream_ack.packets.interval_len()
+                    + self.recovery_ack.packets.interval_len();
+
+                let mut duplicate_threshold = 20;
```

Comment:
How did you arrive at 20?

Reply:
it's _somewhat_ arbitrary but doing some worst-case math the ACK ranges with 20 segments would consume about 20-25% of the packet so it seemed like a good starting point. I didn't want to go too much lower otherwise we end up spamming ACKs too much

Reply:
Maybe a brief comment with your reasoning would be good